README.md

```diff
@@ -1,60 +1,30 @@
-# What is FAST-DNA?
-A design architecture for the web that enables real-time updating of user experiences based on changing inputs. The acronym stands for fast, adaptive, secure, timeless as a design network architecture.
 
-The majority of the effort in building component frameworks goes into the markup, accessibility, localization, and more. Let FAST-DNA do the hard work so you can focus on your design language and user experience.
+# What is FAST-DNA
+FAST-DNA stands for Fast, Adaptive, Secure, and Timeless Design Network Architecture. A set of resources that can be used to accelerate building Web sites and applications.
```

Comment:
Sorry, do you think this reads better?:

"FAST-DNA (Fast, Adaptive, Secure, and Timeless Design Network Architecture) is set of resources that can be used to accelerate building Web sites and applications."

Reply:
I think it could go either way.  I'd like to keep as is for now and reconsider later when Jason's around.  Important that I get this in so i can move CELA request forward.  thx